cellfinder/core/main.py

```diff
@@ -54,6 +55,11 @@ def main(
         Called with the batch number that has just finished.
     detect_finished_callback : Callable[list], optional
         Called after detection is finished with the list of detected points.
+        Number of points processed simultaneously by the neural network.
+    batch_size: int = 64, optional
```

Comment:
```suggestion
    batch_size: int, optional
```
For consistency with the other parameters in the docstring, which don't list their default values.